mirror_macros.h

```diff
@@ -4,7 +4,7 @@
 
 #define MIRROR_CLASS(_class, ...)\
 public:\
-	virtual mirror::Class* getClass() const { return _class::GetClass(); }\
```

Comment:
won't this cause issue compilation on the topmost parent ? does this works if the class is not actually overriding anything ?